packages/flutter/lib/src/semantics/semantics.dart

```diff
@@ -1715,6 +1720,16 @@ class SemanticsProperties extends DiagnosticableTree {
   /// element it is reading, and is separate from input focus.
   final bool? focused;
 
+  /// If non-null, whether the subtree can be focused by accessibility services.
```

Comment:
some where in the doc should mention the behavior when this is false but somewhere in the subtree set this to true. Will that part of the subtree because focusable or it will get override by the parent.